python/nav/ipdevpoll/shadows/__init__.py

```diff
@@ -287,9 +287,8 @@ def prepare(self, containers):
                 self.net_type = net_type
 
     def save(self, containers):
-        if (self._has_no_prefixes(containers) or
-                self._revert_vlan_on_type_change_to_scope(containers) or
-                self._is_type_changed_to_static(containers)):
+        if self._revert_vlan_on_type_change_to_scope(containers) or \
```

Comment:
Which is why PEP-8 suggests this kind of over-indentation for if statements.

I do have sort of a personal preference of storing the results of long boolean expressions in an aptly named variable for use in the actual if-statement in some of these situations :)